src/lib/shopping-event-conversions.js

```diff
@@ -10,50 +11,93 @@ import { getUserId } from './local-storage';
 
 export type EventToFptiInputMapping = (event : Object) => FptiInput;
 
-function getStoredUserIds() : Object {
+const getDefaultFPTIEvent = () => {
   const storedUserIds = getUserId();
-  if (storedUserIds) {
-    return {
-      shopperId: storedUserIds.userId,
-      merchantProvidedUserId: storedUserIds.merchantProvidedUserId
-    };
-  } else {
-    return {};
-  }
-}
 
-function convertShoppingEventToFptiInput(
-  config : Config,
-  event : Object,
-  eventType : EventType
-) : FptiInput {
-  if (!event.user) {
-    event.user = config.user;
-  }
+  const data = {
+    shopperId: storedUserIds ? storedUserIds.userId : undefined,
+    merchantProvidedUserId: storedUserIds ? storedUserIds.merchantProvidedUserId : undefined
+  };
+
+  return data;
+};
+
+function getDefaultFPTIEventFromShoppingEvent(config : Config,
```

Comment:
let's just call it `getFPTIEventFromShoppingEventConverter`, again not a big fan of generic names `generic`, `default`, `info`. I do not think they bring any clarity